.github/dependabot.yml

```diff
@@ -1,3 +1,4 @@
+---
 version: 2
 updates:
   - package-ecosystem: "github-actions"
```

Comment:
Now that we have package.json, should we add `npm` here to keep those tools up to date?

Reply:
It has been added